src/federationparams.cpp

```diff
@@ -159,6 +159,8 @@ CPubKey CFederationParams::ReadAggregatePubkey(const std::vector<unsigned char>&
 
 bool CFederationParams::ReadGenesisBlock(std::string genesisHex)
 {
+    ECCVerifyHandle globalVerifyHandle;
+
```

Comment:
What is this use for?

Reply:
It was needed in unit tests for initialising verification context of signature verification. IT should be in the constructor so I removed from here.